src/server/Html.js

```diff
@@ -34,6 +35,18 @@ export default function Html({
         <meta property="description" content="Team transparency, made easy." />
         <style dangerouslySetInnerHTML={{__html: lookCSSToken}} id={lookConfig.styleElementId} />
         <title>{title}</title>
+        {/* segment.io analytics */}
+        {process.env.SEGMENT_WRITE_KEY &&
```

Comment:
@mattkrick you'll note that I did it as a file include in https://github.com/ParabolInc/action/pull/250/commits/eae4e9627ff26734a60c5c5158c34055498cd449 and slept on it.

The things I didn't quite like were:

-  Importing the pre-computed snipped source into the module global scope was computationally equal to calling the snippet generation function. Either it happens before the server starts or it happens once after the server starts. Same difference AFAIAC.
 - Yet another mystery json file
 - Unless I introduced a branch (a la process.env... &&) there wasn't a great way to omit _some_ tag generation altogeter

I think the approach I've got now is decent.